modules/upgrade-cluster-version-definition.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
```

Comment:
I propose removing the table of parameters and providing an annotated YAML file instead. @jiajliu, @abhinavdahiya, what do you think?

Reply:
I can see how that will be useful to the end users. +1

Reply:
Great! Then I'll pull the table of parameters. I'm happy to add more details to enhance the update instructions if we see gaps. (I'll update the yaml and command outputs when the default upgrade server's working and I can run through all the steps.)

Reply:
I went through the new section about cluster-version at https://github.com/openshift/openshift-docs/blob/ec09b3754165373dbdb8cff44a94c6b557577eae/modules/upgrade-cluster-version-definition.adoc.
lgtm. but one more question is that if you miss some parts of status in the yaml file? 
'''
<--snip-->
    desired:
      image: registry.svc.ci.openshift.org/ocp/release@sha256:06fdf60795e98ceb70ca82550ef2ec3fc1005f27ac3581f46cbebbfcc0b2e25d
      version: 4.0.0-0.nightly-2019-03-28-030453
    history:
    - completionTime: 2019-03-28T05:59:50Z
      image: registry.svc.ci.openshift.org/ocp/release@sha256:06fdf60795e98ceb70ca82550ef2ec3fc1005f27ac3581f46cbebbfcc0b2e25d
      startedTime: 2019-03-28T05:45:07Z
      state: Completed
      version: 4.0.0-0.nightly-2019-03-28-030453
    observedGeneration: 1
    versionHash: wKTJ5oTXq48=
'''

Reply:
After a discussion with Clayton, I'm removing the Cluster Version yaml. There's not a good use case to edit in 4.1.

Reply:
> After a discussion with Clayton, I'm removing the Cluster Version yaml. There's not a good use case to edit in 4.1.

https://github.com/openshift/openshift-docs/blob/40bb56d720556cf3db30d088d39abcad1e25efd1/modules/upgrade-cluster-version-definition.adoc
I notice this section still existed. And several issues:
1. no "namespaces" in metadata?
2. "channel" should be "stable-4.1"
3. there is another condition "RetrievedUpdates"
  - lastTransitionTime: 2019-05-22T07:13:26Z
    status: "True"
    type: RetrievedUpdates

Reply:
You're right. The file is still present, but it's not used in the assembly any more. It doesn't need to be published for 4.1, but we might need it for upgrades for 4.2. I updated the file.